testgrid/config.yaml

```diff
@@ -32,6 +38,12 @@ dashboard_groups:
     - googleoss-kubeflow-gcp-blueprints
   - name: googleoss-kpt-config-sync
     dashboard_names:
-    - googleoss-kpt-config-sync-multi-repo-test-group1
-    - googleoss-kpt-config-sync-multi-repo-test-group2
+    - googleoss-kpt-config-sync-multi-repo-1
```

Comment:
are you sure you want to add so many dashboards?

Reply:
Yes, each test group will have 8 sub tabs. We would like it to be organized by different dashboard groups.